modules/local/ctatsplicing/prepgenomelib/main.nf

```diff
@@ -0,0 +1,45 @@
+process CTATSPLICING_PREPGENOMELIB {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://data.broadinstitute.org/Trinity/CTAT_SINGULARITY/CTAT-SPLICING/ctat_splicing.v0.0.2.simg' :
+        'docker.io/trinityctat/ctat_splicing:0.0.2' }"
+
+    input:
+    tuple val(meta), path(genome_lib)
+
+    output:
+    tuple val(meta), path(genome_lib, includeInputs:true), emit: reference
+    path "versions.yml"                                  , emit: versions
+
+    script:
+    def VERSION = '0.0.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    """
+    wget https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/CANCER_SPLICING_LIB_SUPPLEMENT/cancer_introns.GRCh38.Jun232020.tsv.gz
```

Comment:
Perhaps this one can be another input instead of downloading it? 

```
    input:
    tuple val(meta), path(genome_lib)
    path(cancer_introns_tsv)
```
So that we do not have to change the hardcoded path in the future and we can manage it via config.

Reply:
Good idea!